api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -173,6 +174,9 @@ message ExternalProcessor {
   // :ref:`clear_route_cache <envoy_v3_api_field_service.ext_proc.v3.CommonResponse.clear_route_cache>`
   // field is set in an external processor response.
   bool disable_clear_route_cache = 11;
+
```

Comment:
// [#not-implemented-hide:]
if the implementation is not included in this PR

Reply:
done